File: backend/ChurchApp-1/ChurchApp-1/Models/Event.cs
using System;
using System.Collections.Generic;

namespace ChurchApp_1.Models;

public partial class Event
{
    public int EventId { get; set; }

    public bool IsAttendance { get; set; }

    public string EventName { get; set; } = null!;

    public string? EventDesc { get; set; }

    public string? EventAddress { get; set; }

    public DateTime EventStart { get; set; }

    public DateTime EventEnd { get; set; }

    //public virtual ICollection<SignUp> SignUps { get; set; } = new List<SignUp>();
}

File: frontend/src/types/event.ts
export interface Event {
    eventId: number;
    isAttendance: boolean;
    eventName: string;
    eventDesc: string;
    eventAddress: string;
    eventStart: string;
    eventEnd: string;
    eventTypeId: number | null;
    unitId: number;
    organizationId: number;
    eventType: any | null;
    organization: any | null;
    signUps: any[];
    unit: any | null;
}